src/containers/store/tagline.svelte

```diff
@@ -1,8 +1,23 @@
 <script>
+  import { onMount } from 'svelte';
   import Button from 'ui/button.svelte';
   import Dot from 'ui/dot.svelte';
+  import * as api from '@/utils/api.js';
 
   export let isAdmin;
+  let dotShown = false;
+  
+  onMount(async () => {
+    if (isAdmin) {
+      try {
+        const purchases = await api.json(api.get('/stock_changes/for_review'));
```

Comment:
Perhaps it's worth adding a `?limit=1` param (to backend first) so it doesn't request a lot of unnecessary information since it doesn't store it in a central store or anything and will re-request it on the other page.
But that's probably out of the scope of this PR

Reply:
Yeah, thought about that too, but brushed it off thinking there will never be a surplus of purchases, and since this is asynchronous, it won't slow anything down